publishing.md

```diff
@@ -10,12 +10,12 @@ next: /security
 
 Ways to share your gem code with other users.
 
-* [Introduction](#introduction)
-* [Sharing Source Code](#sharing-source-code)
-* [Serving Your Own Gems](#serving-your-own-gems)
-* [Publishing to RubyGems.org](#publishing-to-rubygemsorg)
-* [Push Permissions on RubyGems.org](#push-permissions-on-rubygemsorg)
-* [Gem Security](#gem-security)
+- [Introduction](#introduction)
+- [Sharing Source Code](#sharing-source-code)
+- [Serving Your Own Gems](#serving-your-own-gems)
+- [Publishing to RubyGems.org](#publishing-to-rubygemsorg)
+- [Push Permissions on RubyGems.org](#push-permissions-on-rubygemsorg)
+- [Gem Security](#gem-security)
```

Comment:
Would you mind undoing the `*` -> `-` change on these lines? Aside from that, I think this is fine. :slightly_smiling_face:

Reply:
Sure @duckinator I can.

Reply:
@duckinator fixed :)